scipy/signal/tests/test_fir_filter_design.py

```diff
@@ -481,6 +481,84 @@ def test_fs_validation(self):
         with pytest.raises(ValueError, match="Sampling.*single scalar"):
             remez(11, .1, 1, fs=np.array([10, 20]))
 
+
+class TestRemezord:
+    """
+    Test examples taken from:
+    http://www.ece.northwestern.edu/local-apps/matlabhelp/toolbox/signal/remezord.html
+
+    with reference values computed in MATLAB.
+    https://github.com/scipy/scipy/pull/20983#discussion_r1649341010
+    """
+
+    def test_bad_args(self):
+        freqs = np.array([0.1, 0.2, 0.3, 0.4])
+        amps = np.array([40, 50, 60])
+        rips = np.array([3, 4, 5])
+        # Nonexistent algorithm
+        assert_raises(ValueError, remezord, freqs, amps, rips, alg="no_alg")
```

Comment:
For new tests please could we use the context manager version
```suggestion
        with assert_raises(ValueError):
            remezord(freqs, amps, rips, alg="no_alg")
```